warehouse/models/staging/payments/elavon/int_elavon__deposit_transactions_deduped.sql

```diff
@@ -0,0 +1,85 @@
+{{ config(materialized='table') }}
+
+WITH
+
+stg_elavon__transactions AS (
+
+  SELECT * FROM {{ ref('stg_elavon__transactions') }}
+
+),
+
+deposit_transactions AS (
+
+  SELECT * FROM stg_elavon__transactions
+  WHERE batch_type = 'D'
+
+),
+
+dedup_deposit_transactions AS (
```

Comment:
I am kind of confused by this CTE, can we comment what it's doing? It kind of seems like it might be equivalent to 

```sql
SELECT * EXCEPT(execution_ts),
execution_ts AS max_ts
FROM deposit_transactions
QUALIFY ROW_NUMBER OVER (PARTITION BY trn_ref_num ORDER BY execution_ts DESC) = 1
```
Which seems a *lot* clearer to me?

Reply:
Updated per convo 10/5: we should just do `QUALIFY DENSE_RANK() OVER (ORDER BY execution_ts DESC)` and do it once for *all* the rows and handle duplicate `trn_ref_num` values downstream if necessary

Reply:
Started a new thread for this